src/views/Search/EmergencyTable/index.tsx

```diff
@@ -113,12 +117,21 @@ function EmergencyTable(props: Props) {
         createListDisplayColumn<EmergencyResult, number>(
             'countries',
             strings.searchEmergencyTableCountry,
-            (emergency) => ({
-                value: emergency.countries,
+            (item) => ({
```

Comment:
We should create a separate column type instead of passing a function to render custom blocks.